src/actions/session.js

```diff
@@ -155,6 +155,23 @@ function postToPortier(server: string, redirect: string): NavigationResult {
   }
 }
 
+export function navigateToOpenID(
+  authFormData: Object,
+  provider: Object
+): NavigationResult {
+  const { origin, pathname } = document.location;
+  const { server } = authFormData;
+  const strippedServer = server.replace(/\/$/, "");
+  const { auth_path } = provider;
```

Comment:
nit: This one's under_score but the next one is camelCase. I.e. `auth_path` and `[stripped]AuthPath`.